parser_library/test/processing/mach_instr_test.cpp

```diff
@@ -254,3 +254,30 @@ TEST(mach_instr_processing, rel_addr_bitmask)
         EXPECT_EQ(context::instruction::mnemonic_codes.at(instr).reladdr_mask.mask(), expected) << instr;
     }
 }
+
+TEST(mach_instr_processing, instr_size)
+{
+    for (const auto& [instr, expected] : std::initializer_list<std::pair<std::string, int>> {
+             { "LARL", 6 },
+             { "LA", 4 },
+             { "CLIJ", 6 },
+             { "BR", 2 },
+             { "DC", 1 },
+         })
+    {
+        EXPECT_EQ(processing::processing_status_cache_key::generate_loctr_len(&instr), expected) << instr;
+    }
+}
+
+TEST(mach_instr_processing, loctr_len_reference)
```

Comment:
Could you think of a test verifying the `L'*` is resolved after resolving opsyn?

Reply:
I think the test above does exactly that and shows a bug. We'll have to wait for the APAR mentioned to replicate the correct new behavior.